tenable/io/v3/vm/scanner_groups/schema.py

```diff
@@ -0,0 +1,12 @@
+'''
+Scanner-groups API Endpoint Schemas
+'''
+
+from marshmallow import Schema, fields
+from marshmallow import validate as v
+
+
+class ScannerGroupsBaseSchema(Schema):
+    name = fields.Str()
+    type = fields.Str(validate=v.OneOf(['load_balancing']))
```

Comment:
I believe the default value is `load_balancing`, it supports other values as well as per description here https://developer.tenable.com/reference#scanner-groups-create

Reply:
It is the only supported value confirmed with Sahil